### ISSUE TITLE
New and existing db postgresql/sqlite3 got BCrypt::Errors::InvalidHash: invalid hash error

### ISSUE THREAD

**Issue by eL-Prova** [Arachni/arachni-ui-web#155]
I try to setup arachni. Got login page but with default credentials it returns Something went wrong.
Looked in log and saw message: `BCrypt::Errors::InvalidHash: invalid hash`

So removed the sqlite3 database and run `arachni_web_task db:setup` but got [WARN] and message if Roles migration has been run. But after that got the same error again.

So tried with postgresql but even that one got the same error. Do anyone know how to solve?

Version: 1.5.1-0.5.12

**Comment by dcabro** [comment]
Same here on Centos

**Comment by dcabro** [comment]
Just found out that arachni is no longer being maintained: https://www.arachni-scanner.com/blog/arachni-is-no-longer-maintained/

**Comment by eL-Prova** [comment]
True, saw it last week too. But its since 28 january 2020 so I dont believe its broken within a month.
I have less experience with ruby yet, but can the solution be to update all gems to make it work?
